lib/util/trip/drive_card.dart

```diff
@@ -70,43 +70,69 @@ class _DriveCardState extends TripCardState<DriveCard> {
     return TripOverview(super.trip!).buildSeatIndicator(context, super.trip!);
   }
 
+  Color pickBannerColor() {
+    if (_drive!.cancelled) {
+      return Theme.of(context).errorColor;
+    } else {
+      return Theme.of(context).own().success;
+    }
+  }
+
   // Notification
   @override
   Widget buildTopRight() {
-    return !fullyLoaded
-        ? const Center(
-            child: SizedBox(
-            height: 24,
-            width: 24,
-          ))
-        : drive!.cancelled
-            ? Icon(Icons.block, color: Theme.of(context).errorColor)
-            : drive!.rides!.any((ride) => ride.status == RideStatus.pending)
-                ? Icon(
-                    Icons.done_all,
-                    color: Theme.of(context).disabledColor,
-                  )
-                : Icon(
-                    Icons.done_all,
-                    color: Theme.of(context).own().success,
-                  );
+    return const SizedBox();
```

Comment:
maybe the default in the TripCardState should just be SizedBox(). So you don't need to implement it when the card is not doing anything with that space